config/make_admin.py

```diff
@@ -2,11 +2,12 @@
 
 import django
 
+from intranet.apps.groups.models import Group
+from intranet.apps.users.models import User
+
```

Comment:
Why is this change happening?

Reply:
I did ./scripts/format.sh and I think it changed that? I can go ahead and delete it, though.